src/components/Q19Division/index.tsx

```diff
@@ -0,0 +1,9 @@
+interface IQ19DivisionProps {
+    value: number;
+    value1: number;
+}
+
+const Q19Division: React.FC<IQ19DivisionProps> = ({ value, value1 }) =>
+    value ? <p>{value / value1}</p> : <p>O divisor não pode ser zero</p>;
+
```

Comment:
Está errado. Se `value1` for zero, vai dar resultado infinito. Você entregou sem testar e isso não deve acontecer nunca